metadata.yaml

```diff
@@ -6,4 +6,17 @@ description: |
   As the next-generation Logstash Forwarder, Filebeat tails logs and quickly
   sends this information to Logstash for further parsing and enrichment or to
   Elasticsearch for centralized storage and analysis.
-series: []
+series: [bionic, focal, jammy]
+subordinate: true
+tags:
+  - filebeat
+requires:
+  beats-host:
+    interface: juju-info
+    scope: container
+  logstash:
+    interface: elastic-beats
+  elasticsearch:
+    interface: elasticsearch
+  kafka:
+    interface: kafka
```

Comment:
hey @esunar and @agileshaw I've pushed some changes which I hope will help with the issues seen. Seems that build no longer works on my workstation, I had to add